tests/kernel/ib_perf_tests.pm

```diff
@@ -0,0 +1,123 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: run InfiniBand performance tests
+#
+# Maintainer: Sebastian Chlad <schlad@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use power_action_utils 'power_action';
+use lockapi;
+use Utils::Backends 'use_ssh_serial_console';
+use ipmi_backend_utils;
+
+
+our $master;
+our $slave;
+
+sub ibtest_slave {
+    # test is utilizing perftest
+    zypper_call('in perftest');
+    barrier_wait('IBTEST_BEGIN');
+    script_run('ib_write_bw');
+    # wait until test is finished
+    barrier_wait('IBTEST_DONE');
+}
+
+sub ibtest_master {
+    my $master             = get_required_var('IBTEST_IP1');
+    my $slave              = get_required_var('IBTEST_IP2');
+
+    # do all test preparations and setup
+    zypper_call('--gpg-auto-import-keys ref');
+    zypper_call('in perftest');
```

Comment:
As you need to intall the same dependencies for master and slave, you should move this to run()